### DISCUSSION TITLE
Offline docs?

### DISCUSSION THREAD

**Discussion by lucasl84** [alpinejs/alpine#4407] [3. Feature ideas]
Hi there!

I was wondering if there's a way to get the AlpineJS documentation for offline browsing.

Thanks,
Lucas.

**Comment by ekwoka** [comment]
I've tried to get it working on dev-docs. Still a WIP as their setup is a bit funky.

Otherwise, the docs are in this repo. So you could just clone it.

**Comment by lucasl84** [comment]
Thanks a lot! I will check it out. My bad for not checking the repo in first place!
